docs/troubleshooting-guide/sync-fingerprint-scan.md

```diff
@@ -1,85 +1,184 @@
 ---
-title: Troubleshooting syncs and scans
+title: Troubleshooting syncs, scans, and fingerprinting
 ---
 
-# Troubleshooting syncs and scans
+# Troubleshooting syncs, scans, and fingerprinting
 
-Metabase needs to know what's in your database in order to show tables and fields, populate dropdown menus, and suggest good visualizations, but loading all the data would be very slow (or simply impossible if you have a lot of data). It therefore does three things:
+First, check if your data is outdated because of browser caching:
 
-1. Metabase periodically asks the database what tables are available, then asks which columns are available for each table. We call this *syncing*, and it happens [hourly or daily][sync-frequency] depending on how you've configured it. It's very fast with most relational databases, but can be slower with MongoDB and some [community-built database drivers][community-db-drivers].
+1. Clear your browser cache.
+2. Refresh your Metabase page.
+3. Open your Metabase page in an incognito window.
 
-2. Metabase *fingerprints* the column the first time it synchronizes. Fingerprinting fetches the first 10,000 rows from each column and uses that data to guesstimate how many unique values each column has, what the minimum and maximum values are for numeric and timestamp columns, and so on. Metabase only fingerprints each column once, unless the administrator explicitly tells it to fingerprint the column again, or in the rare event that a new release of Metabase changes the fingerprinting logic.
+Once you've confirmed that you're looking at a non-cached view of your tables and columns, tag your database admin for help with troubleshooting:
 
-3. A *scan* is similar to fingerprinting. Metabase will scan a database by default every 24 hours (though you can configure Metabase to run a scan less frequently, or disable scanning entirely). When you set a field to "A list of all values" in the [Data Model](../data-modeling/metadata-editing.md), which is used to display options in dropdown menus, scanning looks at the first 1,000 distinct records (ordered ascending). For each field scanned, Metabase stores only the first 100 kilobytes of text. If more values exist, Metabase displays the stored values in the dropdown menus, and only triggers a database search query to look for more values when people type in the search box for that filter widget.
+- **Syncs**, if your tables or columns are missing, or your column data types are wrong.
+- **Scans**, if your column _values_ are wrong (for example, if your filter dropdown menus contain the wrong values).
+- **Fingerprinting**, if you've triggered a manual scan, but the changes aren't taking effect.
 
-## Metabase can't sync, fingerprint, or scan
+## Initializing a sync
```

Comment:
There really isn't any much difference between this section and "Syncing new or updated tables and views", so might as well merge them together.